Find a functionally equivalent code:android.view.HardwareLayer.destroy:COMMENT
<android.view.HardwareLayer: void destroy()>
Method Modifier: public      default     abstract    
Method Modifier: No          Yes         Yes         
Method Updated:  Yes         No          No          
Comment:/**
 * Destroys resources without waiting for a GC.
 */

@@ -1 +1,10 @@
-
+{
+    if (!isValid()) {
+        // Already destroyed
+        return;
+    }
+    mRenderer.onLayerDestroyed(this);
+    mRenderer = null;
+    mFinalizer.release();
+    mFinalizer = null;
+}

